rdmo/projects/assets/js/actions/projectsActions.js

```diff
@@ -29,7 +31,21 @@ export function fetchProjectsSuccess(projects) {
 }
 
 export function fetchProjectsError(error) {
-  return {type: FETCH_PROJECTS_ERROR, error}
+  return function(dispatch) {
+    if (error.response && error.response.status === 400 && error.body && error.body.catalog) {
+      dispatch(configActions.deleteConfig('params.catalog'))
+      dispatch(fetchAllProjects())
+      dispatch({
```

Comment:
I think I would not dispatch an error here, or I missed something. If we add the errors at some point to the user interface this could cause problems.

Reply:
It's not displayed in the user interface, but in the redux-logger / console

Reply:
I think it should be removed, in the end, the error is solved, nothing needs to be done as an reaction. The action is not needed for logging, it is meant do be used in the reducer. In principle, we could also remove the code from the reduce if do not do anything with the (possible) error anyway. But I would keep it for now, maybe we will use it in the future.